src/pytorch_lightning/loggers/tensorboard.py

```diff
@@ -40,6 +38,13 @@
 log = logging.getLogger(__name__)
 
 _TENSORBOARD_AVAILABLE = RequirementCache("tensorboard")
+_TENSORBOARDX_AVAILABLE = RequirementCache("tensorboardX")
+if TYPE_CHECKING:
+    # assumes at least one will be installed when type checking
+    if _TENSORBOARD_AVAILABLE:
```

Comment:
this is opposite what @lantiga said in https://github.com/Lightning-AI/lightning/pull/15728#issuecomment-1322363710 :otter: